corehq/apps/change_feed/tests/test_pillow.py

```diff
@@ -122,6 +128,16 @@ def test_mismatched_rev(self):
                 )
             )
 
+        with self.assertRaises(DocumentMismatchError):
```

Comment:
there's one that ensures that the same doesn't raise an exception `good_metadata` which checks that the equality check is working. it implies that the `2-me` would also work, but in the future if this ever changes it could be good to have. will add an additional case